regions/io/stcs/example_usage.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
```

Comment:
this whole file is not needed; it's basically a documentation example.  So I'll cut it.